app.py

```diff
@@ -3,6 +3,7 @@
 import db_management
 import login_utils
 import signup_utils
+from signup_utils import SignupResults
 
 app = Flask(__name__)
 app.secret_key = b'_5#y2L"F4Q8z\n\xec]/'
```

Comment:
You never keep secret keys in public github repositories!!!!!!!!!

Reply:
i made the secret key a user variable (on my local machine) and is now hidden